website/src/blogs/2017-05-23-multi-node-kubernetes-1-6-cluster-provisioning-made-easy-using-sandbox-vagrant-box.md

```diff
@@ -0,0 +1,73 @@
+---
+
+### Background
+
+Working on OpenEBS, a containerized storage for containers which is orchestrated by Kubernetes, most of our tasks, be it development, testing and demo require us to setup and modify the nodes in Kubernetes cluster. In addition, a multi-node cluster is a must as we go beyond the initial development and testing, to explore the high availability, scale, performance, and upgrade aspects.
+
+While *minikube *and *minishift* provide an easy way to setup Kubernetes single node cluster — for multi-node cluster the fastest ways to get going are usually cloud or hosted solutions. ***kubeadm*** is the closest we can get to easily setup a cluster, but since it is still in alpha, we keep running into some issues like — [kubadm init v1.6.1 fails](https://github.com/kubernetes/kubeadm/issues/226)
+
+*An ideal solution for a developer would be a Kubernetes Sandbox. A sandbox that can be easily setup on a laptop and can work on the move (without net connectivity.) This Sandbox should be shielded from the different API or CLI changes that happen with the frequent releases of kubeadm and Kubernetes.*
+
+We have used Vagrant, VirtualBox, and Atlas to do just that.
+![Vagrant, VirtualBox and Atlas](https://cdn-images-1.medium.com/max/800/1*7kkviZOwgh8ePDYRjFX0mQ.png)
+
+## Try It! It is Easy and Quick!
+
+Once you have Vagrant (1.9.1 or higher) and VirtualBox (5.1.14 or higher) installed on your laptop/machine, just do the following:
+
+Step 1: Download the Vagrantfile from [OpenEBS Github](https://raw.githubusercontent.com/openebs/openebs/master/k8s/lib/vagrant/test/k8s/1.6/Vagrantfile)
+Step 2: Run **vagrant up**
+
+Detailed instructions can be found [here](https://github.com/openebs/openebs/tree/master/k8s/lib/vagrant/test/k8s/1.6).
+
+The above two steps will provision the following:
+
+- Ubuntu VM with Kubernetes Master (kubemaster-01)
+- Ubuntu VM with Kubernetes Minion (kubeminion-01) associated with (kubemaster-01)
+- Setup *weave *as pod network
+- Setup the kubectl credentials ( admin.conf) on kubemaster-01
+- Sample Kubernetes pod YAML files are located on (kubemaster-01) under the directory ( *~/demo/k8s/spec/* )
+
+In addition to the above, the following OpenEBS provisioning tasks are also performed.
+
+- Install OpenEBS iSCSI FlexVolume Driver on the kubeminion-01
+- Ubuntu VMs installed with OpenEBS Maya Master and OpenEBS Storage Hosts. (If you don’t want to use the storage, you can skip the installation of these VMs. Check the customization steps below).
+
+## Customizing the Kubernetes Sandbox
+
+The above instructions include setting up Kubernetes and OpenEBS as well, but you can easily customize the Vagrantfile to skip installation of OpenEBS by prefixing ENV variables before the vagrant command as follows:
+
+    MM_NODES=0 MH_NODES=0 vagrant up
+
+Some of the configuration options available are:
+
+- KM_CPU — Number of CPUs for minion (*default 2*)
```

Comment:
```suggestion
- KM_CPU — Number of CPUs for minion *(default 2)*
```

Reply:
Do the curly braces need to be itallic?

Reply:
It should be consistent with other items in the list.